frontend/src/components/VErrorSection/VErrorImage.vue

```diff
@@ -71,12 +73,13 @@ export default defineComponent({
 
 <style scoped>
 ::v-deep(.attribution) {
-  @apply text-dark-charcoal-70;
+  @apply mt-4 text-sr text-dark-charcoal-70;
 }
-::v-deep(a) {
+::v-deep(.attribution a) {
   @apply text-current underline;
 }
-::v-deep(img) {
-  @apply opacity-70; /* to match the text color */
+/* license icons should match the text color */
+::v-deep(.attribution img) {
```

Comment:
Previous declaration was causing the large error images to have lower opacity, too. Designs only have lower opacity for the license icons in the attribution sentence.